Mathlib/Algebra/Group/End.lean

```diff
@@ -661,6 +661,16 @@ theorem one_def : (1 : MulAut M) = MulEquiv.refl _ :=
 theorem inv_def (e₁ : MulAut M) : e₁⁻¹ = e₁.symm :=
   rfl
 
+@[simp]
+theorem inv_symm (e : MulAut M) : e⁻¹.symm = e := rfl
+
+@[simp]
+theorem symm_inv (e : MulAut M) : (e.symm)⁻¹ = e := rfl
+
+@[simp]
+theorem inv_apply (e : MulAut M) (m : M) : e⁻¹ m = e.symm m := by
+  rw [inv_def]
```

Comment:
Could you put this below `one_apply` to keep the `mul`,`one`,`inv` order used above (or any order you like, consistently)

Reply:
done.